test/unit/org/apache/cassandra/cql3/CQLTester.java

```diff
@@ -1943,6 +1949,132 @@ protected void assertRowCountNet(ResultSet r1, int expectedCount)
         Assert.assertEquals(String.format("expected %d rows but received %d", expectedCount, actualRowCount), expectedCount, actualRowCount);
     }
 
+    public abstract static class CellValidator
```

Comment:
the store changes depending on the junit execution order (which isn't fixed) so needed a way to say "any" while still validating its a non-null int returned...  once I had this I found more usages such as the status being multiple possible ones!